applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -95,6 +97,15 @@ def __init__(self, model, custom_settings):
 
         self._bfecc_convection = self.settings["bfecc_convection"].GetBool()
 
+        self._eulerian_error_compensation = False
+        if (self.settings.Has("eulerian_error_compensation")):
+            self._eulerian_error_compensation = self.settings["eulerian_error_compensation"].GetBool()
+
+        self._levelset_dt_factor = 1.0 # this is used to identify the splitting of LS convection (Strang splitting idea)
+        if (self.settings.Has("split_levelset")):
```

Comment:
Same comment.

Reply:
I'd also rename `split_levelset` to `strang_splitting`, which I think is more self-descriptive for the user.

Reply:
If you want to keep it flexible, we should expose the `levelset_dt_factor` to the user.

Reply:
Let's keep dt_factor = 1.0 or 0.5, and do not expose it in the settings (for now).

Reply:
Then we should expose the bool `split_levelset` (or something like `strang_splitting`, `levelset_splitting` ,...)

Reply:
It is currently: "split_levelset" : false/true
Will be changed to "levelset_splitting" : false/true